raw-svo.defs.lua

```diff
@@ -210,7 +210,10 @@ defs_data = pl.OrderedMap {}
       end
 
       return true
-    end}) -- added in xml w/ conf.gagbreath
+    end,
+    def = [[You are holding your breath.]],
+    ondef = function() return "" end, --workaround to make it show up in defense lists...
```

Comment:
Yeah, this is a strange one.

Upon first analysis, I thought this was the only way to mark a defense with an X if it's not in the defc table.

After another inspection I'm not even sure how I traced it down to this... :cry: